example/wildcat.config.js

```diff
@@ -49,32 +49,87 @@ var wildcatConfig = {
         // https://www.npmjs.com/package/gelf-pro#configuration
         // graylog: {},
 
-        // Enable http2
-        http2: false,
-
-        // Enable https
-        https: true,
-
-        // Default port number
-        port: 3000,
-
-        // A key/value of urls to proxy
-        // e.g. /static -> http://example.com/static
-        proxies: {
-            "/static": "http://example.com/static"
-        },
-
         // Directory to output compiled JavaScript modules
         publicDir: "public",
 
-        // Path to the entry config file relative to the project root
-        entry: "public/main.js",
-
-        // Path to the server renderer. This can be a jspm package or a relative path
-        renderHandler: "react-wildcat-handoff/server",
-
         // Path to your source JavaScript files
-        sourceDir: "src"
+        sourceDir: "src",
+
+        // config options for the app server
+        appServer: {
+            // Enable http2
+            http2: true,
```

Comment:
Right now, looks like the priority order is `http2: true`, then `https: true`, or neither is defined so assume `http`.  Might be easier to config if you changed this to a single string instead of 3 vars:  `protocol: "http2" | "https" | "http"`